pkgs/applications/networking/instant-messengers/armcord/default.nix

```
@@ -131,7 +131,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Lightweight, alternative desktop client for Discord";
-    homepage = "https://github.com/ArmCord/ArmCord";
+    homepage = "https://armcord.app";
```

```suggestion
    homepage = "https://armcord.app";
    downloadPage = "https://github.com/ArmCord/ArmCord";
```
> I was curious if there was another meta attribute for this. Feel like it's nicer to have the actual source code link instead of a front facing website.

You may want to add `meta.downloadPage`.